RecoLocalTracker/SiPixelClusterizer/plugins/SiPixelRawToClusterGPUKernel.cu

```diff
@@ -692,12 +690,14 @@ namespace pixelgpudetails {
 
     findClus<<<blocks, threadsPerBlock, 0, stream.id()>>>(
                  moduleInd_d,
-                 xx_d, yy_d, adc_d,
+                 xx_d, yy_d,
                  moduleStart_d,
                  clusInModule_d, moduleId_d,
                  clus_d,
                  wordCounter
     );
+    cudaDeviceSynchronize();
```

Comment:
Is this `cudaDeviceSynchronize()`  leftover from the debugging?

Reply:
yes...